src/v2/openapi.yaml

```diff
@@ -0,0 +1,740 @@
+        - unknown_dependencies_count
+        - unknown_dependencies
+        - analyzed_direct_dependencies_count
+        - transitive_count
+        - vulnerable_transitives_count
+        - license_analysis
+        - distinct_licenses
+        - stack_license_conflict
+        - dependencies
+        - total_licenses
+        - recommendation_ready
+      discriminator:
+        propertyName: registration_status
+        mapping:
+          registerd: StackAnalysesResultForRegisteredUser
+          freetier: StackAnalysesResultForFreeTier
+      properties:
+        _audit:
+          $ref: '#/components/schemas/Audit'
+        uuid:
+          type: string
+          format: uuid          
+        external_request_id:
+          type: string
+          example: 229e7da76708fe374d8c10fa752e72989f      
+        registration_status:
+          $ref: '#/components/schemas/RegistrationStatus'
+        manifest_file_path:
+          type: string
+          example: pom.xml
+        manifest_name:
+          type: string
+          example: dependencies.txt
+        ecosystem:
+          $ref: '#/components/schemas/Ecosystem'
+        unknown_dependencies_count:
+          type: integer
+          example: 2
+        unknown_dependencies:
+          type: array
+          items:
+            $ref: '#/components/schemas/Package'
+          example:
+            - name: rocket-db
+              version: 2020.1.alpha
+            - name: jsonp
+              version: 0.0.1
+        analyzed_direct_dependencies_count:
+          type: integer
+          example: 25              
+        transitive_count:
+          type: integer
+          default: -1
+          example: 11
+        vulnerable_transitives_count:
+          type: integer
+          default: -1
+          example: 1
+        license_analysis:
+          $ref: '#/components/schemas/LicenseAnalysis'
+        distinct_licenses:
+          type: array
+          items:
+            type: string
+        stack_license_conflict:
+          type: boolean
+        dependencies:
+          type: array
+          items:
+            $ref: '#/components/schemas/PackageWithTransitives'
+        total_licenses:
+          type: integer
+          example: 15
+        recommendation_ready:
+          type: boolean
+      description: Stack analyses result data
+
+    StackAggregatorResultForRegisteredUser:
+      title: StackAggregatorResultForRegisteredUser
+      allOf:
+      - $ref: '#/components/schemas/StackAggregatorResult'      
+      - type: object
+        required:
+          - analyzed_direct_dependencies
+          - vulnerable_transitives
+        properties:
+          analyzed_direct_dependencies:
+            type: array
+            items:
+              $ref: '#/components/schemas/PackageDetailsForRegisteredUser'
+            description: |
+              All direct dependencies details regardless of it's vulnerability status
+          vulnerable_transitives:
```

Comment:
This should not be required.

Reply:
As I mentioned, `analyzed_direct_dependencies` field includes details about direct dependency regardless of it's vulnerability status. However `vulnerable_transitives` includes transitive details only if it has vulnerability. 

So simply including all corresponding values of `vulnerable_transitives` within `Package` model would confuse the consumer of this API.

@SriKrishnaPaparaju , Please comment.

Reply:
As discussed, now it is part of PackageDetails.